docs/build-docs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+rm -rf build/
+
+# Move theming elements into the docs folder
+cp -R pytket-docs-theming/_static .
+cp -R pytket-docs-theming/quantinuum-sphinx .
+cp pytket-docs-theming/conf.py .
+
+# Get the name of the project
+EXTENSION_NAME="$(basename "$(dirname $(pwd))")"
+
+# Correct github link in navbar
+sed -i '' 's#CQCL/tket#CQCL/'$EXTENSION_NAME'#' _static/nav-config.js
```

Comment:
This line is actually not needed as we no longer have the github link in the navbar